diff -Nur wwwcount2.6.orig/Count-config wwwcount2.6/Count-config
--- wwwcount2.6.orig/Count-config	Sat Aug 21 16:33:08 1999
+++ wwwcount2.6/Count-config	Sun Jan  7 01:23:11 2001
@@ -20,7 +20,7 @@
 trap 'echo '';echo "Interrupt detected..exiting";rm -f "./Config.tmpl" 2>/dev/null;rm -f "./.alfcf" ;rm -f "./strict" 2>/dev/null;echo '';exit 1' 1 2 3 15
 
 Tmpl="./Config.tmpl"
-CgiBinDir="/usr/local/apache/cgi-bin"
+CgiBinDir="/home/services/httpd/cgi-bin"
 
 if [ ! -f "./VERSION" ] 
 then
@@ -35,22 +35,22 @@
 
 AUTHOR="ma_muquit@fccc.edu"
 Version=`cat ./VERSION`
-Basedir="wwwcount$Version"
+Basedir="wwwcount-$Version"
 ConfigH="src/config.h"
-BaseDir="/usr/local/etc/Counter"
-ConfigDir="$BaseDir/conf"
-ConfFile="count.cfg"
+BaseDir="/usr/lib/wwwcount"
+ConfigDir="/etc"
+ConfFile="wwwcount.cfg"
 DigitDir="$BaseDir/digits"
-DATABASE_DIR="$BaseDir/db"
+DATABASE_DIR="/var/lib/wwwcount/db"
 COUNT_DATABASE="Countdb"
 AUTH_DATABASE="Authdb"
 DB_LOCKFILE="Count.lck"
 TT_FONT_DIR="$BaseDir/fonts"
-DataDir="$BaseDir/data"
-LogDir="$BaseDir/logs"
+DataDir="/var/lib/wwwcount/data"
+LogDir="/var/lib/wwwcount/log"
 ext=`echo $Version | sed 's/\./_/g'`
-LogFile="error.log"
-Visitor_LogFile="visitor.log"
+LogFile="wwwcount-error"
+Visitor_LogFile="wwwcount-visitor"
 RgbFile="./data/rgb.txt"
 
 
@@ -98,122 +98,26 @@
 ##
 # initial screeen
 ###
-cat << EOSI
-    Welcome to the configuration procedure of Count $Version
-    ---------------------------------------------------
-
-    o You must know where your system keeps CGI programs (cgi-bin directory)
-      It is necessary to generate the install program.
-      This directory must exist. If this directory does not exist, the
-      configuration procedure will Abort!
-
-    o You have to decide a directory, where you will keep all counter related
-      stuff. This directory will have other directories inside. Default
-      values will be supplied, press Return key to accept the default value.
-      
-      Try to Accept the default value, it will make your life much easier, 
-      because not much error checking is done to verify input.
-
-    During installation, the directories will be created for you
-    if they do not exist and if you have the permission to do so.
-    ++
-EOSI
 
 
 ###
-echo $n "Continue [y|n]$c? "
-read answer
-
-if [ .$answer = . ]
-then
-    answer=x
-fi
-    until [ $answer = "y" -o $answer = "Y" -o $answer = "n" -o $answer = "N" ]
-    do
-        echo $n "Continue [y|n]$c? "
-        read answer
-            if [ .$answer = . ]
-            then
-                answer=x
-            fi
-    done
-
-    case $answer in
-        y|Y)
 ####################
 # Ask for cgi-bin     --starts
 ###
-cat <<EOB
-
-    You need to enter the full path of the directory where you system
-    keeps the CGI programs. 
-
-EOB
 # it used to be dierctory as you see :)
 # thanks to ashley@photocraftlab.com 
 #echo $n "*cgi-bin dierctory [$CgiBinDir]:$c "
-
-echo $n "*cgi-bin directory [$CgiBinDir]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    CgiBinDir=$newpath
-fi
 # check if the directory exist
-
-if [ ! -d $CgiBinDir ]
-then
-cat <<EOCG  
-
-    WARNING:
-    Directory "$CgiBinDir" does not exist! If you do not know where your
-    system keeps CGI programs, find it out first. If you did not configure
-    your http server yet, please do so and come back!
-
-    Presss Control-C to exit, any key to continue
-
-EOCG
-    read foo
-fi
-
 ####################
 # Ask for cgi-bin     --ends
 ###
 ####################
 # Ask for basedir     --starts
 ###
-cat <<EOB
-
-    You need to enter the base directory of the counter related stuff.
-    It must be a absolute path.
-
-EOB
-echo $n "*Base directory [$BaseDir]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    BaseDir=$newpath
-    ConfigDir=$BaseDir/conf
-    DigitDir=$BaseDir/digits
-    DATABASE_DIR=$BaseDir/db
-    TT_FONT_DIR=$BaseDir/fonts
-    DataDir=$BaseDir/data
-    LogDir=$BaseDir/logs
-fi
 
 #
 # base directory must start with /
 #
-if echo "$BaseDir" | grep -s "^/" >/dev/null 2>&1
-then
-    x="ok"
-else
-    echo 
-    echo "Base directory must be a absolute path ...exiting"
-    echo "please try again"
-    echo 
-    exit 0
-fi
 
 ####################
 # Ask for basedir     --ends
@@ -221,17 +125,6 @@
 ####################
 # Ask for configdir     --starts
 ###
-cat <<EOB
-
-    You need to enter the name directory of the configuration file.
-
-EOB
-echo $n "* Config directory [$ConfigDir]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    ConfigDir=$newpath
-fi
 
 ####################
 # Ask for configdir     --ends
@@ -239,33 +132,6 @@
 ####################
 # Ask for configfile     --starts
 ###
-cat <<EOB
-
-    You need to enter the name of the configuration file. Note, it
-    is not the path of the file, just the name. No / is allowed in the
-    name.
-    This file contains the information about
-
-        o if you want to ignore access from certain hosts
-        o host access authentication
-
-    You will create this file later by running the program "Gen_conf".
-
-EOB
-echo $n "* Name of the configuration file [$ConfFile]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    ConfFile=$newpath
-fi
-if echo "$ConfFile" | grep -s "^/" >/dev/null 2>&1
-then
-    echo 
-    echo "ConfFile name can not have / in it"
-    echo "exiting.. please try again"
-    echo 
-    exit 0
-fi
 
 ####################
 # Ask for configfile     --end
@@ -273,17 +139,6 @@
 ####################
 # Ask for datadir     --starts
 ###
-cat <<EOB
-              
-    You need to enter the directory of the counter data file.
-    
-EOB
-echo $n "*Data directory [$DataDir]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    DataDir=$newpath
-fi  
 ####################
 # Ask for datadir     --ends
 ###
@@ -296,17 +151,6 @@
 ####################
 # Ask for digitdir     --starts
 ###
-cat <<EOB   
-
-    You need to enter the directory of digits
-
-EOB
-echo $n "*Digits directory [$DigitDir]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    DigitDir=$newpath
-fi
 
 ####################
 # Ask for digitdir     --ends
@@ -315,17 +159,6 @@
 ####################
 # Ask for logdir     --starts
 ###
-cat <<EOB   
-
-    You need to enter the directory of the Log file.
-
-EOB
-echo $n "*Log directory [$LogDir]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    LogDir=$newpath
-fi
 ####################
 # Ask for logdir     --ends
 ###
@@ -333,28 +166,6 @@
 ####################
 # Ask for logfile     --starts
 ###
-cat <<EOB
-
-    You need to enter the name of the error log file. Note, it just a name, 
-    not the path. No / is allowed in the name.  This file hold the error 
-    messages of the counter. It also logs if someone tried to access your 
-    counter remotely.
-
-EOB
-echo $n "* Name of the log file [$LogFile]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    LogFile=$newpath
-fi
-if echo "$LogFile" | grep -s "^/" >/dev/null 2>&1
-then
-    echo 
-    echo "LogFile can not have a / in the name"
-    echo "exiting.. please try again"
-    echo 
-    exit 0
-fi
 
 ####################
 # Ask for logfile     --ends
@@ -363,44 +174,11 @@
 ####################
 # Ask for visitor info logfile     --starts
 ###
-cat <<EOB
-
-    You need to enter the name of the visitor log file. Note, it just a name, 
-    not the path. No / is allowed in the name.  This file hold the error 
-    messages of the counter. It also logs if someone tried to access your 
-    counter remotely.
-
-EOB
-echo $n "* Name of the log file [$Visitor_LogFile]:$c "
-read newpath
-if [ .$newpath != . ]
-then
-    Visitor_LogFile=$newpath
-fi
-if echo "$Visitor_LogFile" | grep -s "^/" >/dev/null 2>&1
-then
-    echo 
-    echo "Visitor_LogFile can not have a / in the name"
-    echo "exiting.. please try again"
-    echo 
-    exit 0
-fi
 
 ####################
 # Ask for visitor info logfile     --ends
 ###
 
-        ;;
-        n|N)
-            echo "Exiting..Better luck next time!"
-            exit 1
-        ;;
-
-        *)
-            echo ""
-            echo "Illegal choice"
-     esac
-
 echo ""
 echo "Your configuration:"
 echo "----------------------------------------------------------------"
@@ -420,26 +198,6 @@
     echo "Visitor log file      = $Visitor_LogFile"
 echo "----------------------------------------------------------------"
 
-echo $n "Everything looks OK [y|n]$c? "
-read answer
-
-if [ .$answer = . ]
-then
-    answer=x
-fi
-    until [ $answer = "y" -o $answer = "Y" -o $answer = "n" -o $answer = "N" ]
-    do
-        echo $n "Everything looks ok [y|n]$c? "
-        read answer
-            if [ .$answer = . ]
-            then
-                answer=x
-            fi
-    done
-
-    case $answer in
-        y|Y)
-        echo "Great! creating header file $ConfigH"
 ###
 # create header file --starts
 ###
@@ -622,13 +380,4 @@
 ##
 # create template for install program --ends
 ##
-        ;;
-        n|N)
-            echo "Exiting..Better luck next time!"
-        ;;
-
-        *)
-            echo ""
-            echo "Illegal choice"
-     esac
 # End of configuration script
diff -Nur wwwcount2.6.orig/Count-install wwwcount2.6/Count-install
--- wwwcount2.6.orig/Count-install	Sat Aug 21 17:25:04 1999
+++ wwwcount2.6/Count-install	Sun Jan  7 01:23:11 2001
@@ -13,7 +13,7 @@
 # Nov 24, 1995
 ##
 ver=`cat ./VERSION`
-Basedir="wwwcount$ver"
+Basedir="wwwcount-$ver"
 INSTALL_PROG="./install-sh"
 Tmpl="./Config.tmpl"
 Binary="./bin/Count.cgi"
@@ -74,7 +74,7 @@
 then
 cat<<EOT
     
-    The varaible template file "$Tmpl" does not exist! This file
+    The variable template file "$Tmpl" does not exist! This file
     is created in the configuration procedure. Apparently your 
     configuration procedure did not succeed!
 
@@ -132,7 +132,7 @@
 
 cat <<EOC
 
-    The configuration file $CONFIG_FILR is not created!
+    The configuration file $CONFIG_FILE is not created!
     please create it first by running the script ./Gen-conf
 
     Aborting installation!
diff -Nur wwwcount2.6.orig/build wwwcount2.6/build
--- wwwcount2.6.orig/build	Fri Sep 17 21:28:43 1999
+++ wwwcount2.6/build	Sun Jan  7 01:24:12 2001
@@ -204,46 +204,6 @@
 
 
 
-$rm config.cache
-
-# temporariy send stdout and stderr to /dev/null, first save them
-    exec 8<&1
-    exec 9<&2
-
-    exec 1>/dev/null 2>&1
-        (cd src; make clean)
-
-        for i in $common_libs; do
-            (cd ./libs/$i; make clean)
-        done
-
-        for i in $db_libs; do
-            (cd ./libs/$i; make clean)
-        done
-
-        for i in $common_utils; do
-            (cd ./utils/$i; make clean)
-        done
-
-        for i in $db_utils; do
-            (cd ./utils/$i; make clean)
-        done
-# restore stdout
-    exec 1<&8
-    exec 2<&9
-
-if test x"$all" = x"yes"; then
-    config_options='';
-fi
-
-#echo "config options=$config_options"
-
-./configure $config_options
-status=$?
-
-if test $status -ne 0; then
-    exit
-fi
 
 ##
 ## make common libraries
diff -Nur wwwcount2.6.orig/libs/bigplus/Makefile.in wwwcount2.6/libs/bigplus/Makefile.in
--- wwwcount2.6.orig/libs/bigplus/Makefile.in	Sun Aug  8 19:20:30 1999
+++ wwwcount2.6/libs/bigplus/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -16,7 +16,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1
-CFLAGS= -O $(DEFINES)
+CFLAGS= @CFLAGS@ $(DEFINES)
 
 SRCS = bigplus.c
 OBJS = bigplus.o
diff -Nur wwwcount2.6.orig/libs/combine/Makefile.in wwwcount2.6/libs/combine/Makefile.in
--- wwwcount2.6.orig/libs/combine/Makefile.in	Sat Aug 21 23:08:28 1999
+++ wwwcount2.6/libs/combine/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -21,7 +21,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -D__USE_FIXED_PROTOTYPES__ -DSYS_UNIX=1
-CFLAGS= -O $(DEFINES)
+CFLAGS= @CFLAGS@ $(DEFINES)
 
 SRCS = crbasei.c allocim.c readgif.c rddata.c dstraimg.c\
        readblk.c lzwdec.c syncimg.c duplicim.c writegif.c lsbfwrs.c\
diff -Nur wwwcount2.6.orig/libs/ldbm/Makefile.in wwwcount2.6/libs/ldbm/Makefile.in
--- wwwcount2.6.orig/libs/ldbm/Makefile.in	Sun Aug  8 19:20:30 1999
+++ wwwcount2.6/libs/ldbm/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -51,7 +51,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1 $(LDBMBACKEND)
-CFLAGS= -O $(DEFINES)
+CFLAGS= @CFLAGS@ $(DEFINES)
 
 SRCS = ldbm.c
 OBJS = ldbm.o
diff -Nur wwwcount2.6.orig/libs/mcfg/Makefile.in wwwcount2.6/libs/mcfg/Makefile.in
--- wwwcount2.6.orig/libs/mcfg/Makefile.in	Sun Aug  8 19:20:30 1999
+++ wwwcount2.6/libs/mcfg/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -14,7 +14,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1
-CFLAGS= -O $(DEFINES)
+CFLAGS= @CFLAGS@ $(DEFINES)
 
 SRCS = mcfg.c
 OBJS = mcfg.o
diff -Nur wwwcount2.6.orig/libs/mutils/Makefile.in wwwcount2.6/libs/mutils/Makefile.in
--- wwwcount2.6.orig/libs/mutils/Makefile.in	Tue Aug 31 20:14:02 1999
+++ wwwcount2.6/libs/mutils/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -14,7 +14,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1
-CFLAGS= -O $(DEFINES)
+CFLAGS= @CFLAGS@ $(DEFINES)
 
 SRCS = string.c mutils.c
 OBJS = string.o mutils.o
diff -Nur wwwcount2.6.orig/libs/sdbm/Makefile.in wwwcount2.6/libs/sdbm/Makefile.in
--- wwwcount2.6.orig/libs/sdbm/Makefile.in	Thu Aug 26 21:06:40 1999
+++ wwwcount2.6/libs/sdbm/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -15,7 +15,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1 
-CFLAGS= -O $(DEFINES)
+CFLAGS= @CFLAGS@ $(DEFINES)
 
 SRCS = sdbm.c sdbm_hash.c sdbm_pair.c
 OBJS = sdbm.o sdbm_hash.o sdbm_pair.o
diff -Nur wwwcount2.6.orig/libs/sll/Makefile.in wwwcount2.6/libs/sll/Makefile.in
--- wwwcount2.6.orig/libs/sll/Makefile.in	Sun Aug  8 19:20:30 1999
+++ wwwcount2.6/libs/sll/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -14,7 +14,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1
-CFLAGS= -O $(DEFINES)
+CFLAGS= @CFLAGS@ $(DEFINES)
 
 SRCS = sll.c
 OBJS = sll.o
diff -Nur wwwcount2.6.orig/utils/dumpdb/Makefile.in wwwcount2.6/utils/dumpdb/Makefile.in
--- wwwcount2.6.orig/utils/dumpdb/Makefile.in	Sat Sep  4 05:16:18 1999
+++ wwwcount2.6/utils/dumpdb/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -20,7 +20,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1 $(LDBMBACKEND)
-CFLAGS= -O $(DEFINES) $(LDBM_INCLUDES)
+CFLAGS= @CFLAGS@ $(DEFINES) $(LDBM_INCLUDES)
 
 SRCS = dumpdb.c
 OBJS = dumpdb.o
diff -Nur wwwcount2.6.orig/utils/editdb/Makefile.in wwwcount2.6/utils/editdb/Makefile.in
--- wwwcount2.6.orig/utils/editdb/Makefile.in	Sat Sep  4 05:16:26 1999
+++ wwwcount2.6/utils/editdb/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -20,7 +20,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1 $(LDBMBACKEND)
-CFLAGS= -g $(DEFINES) $(LDBM_INCLUDES)
+CFLAGS= @CFLAGS@ $(DEFINES) $(LDBM_INCLUDES)
 
 SRCS = editdb.c
 OBJS = editdb.o
diff -Nur wwwcount2.6.orig/utils/extdgts/Makefile.in wwwcount2.6/utils/extdgts/Makefile.in
--- wwwcount2.6.orig/utils/extdgts/Makefile.in	Sat Sep  4 05:16:34 1999
+++ wwwcount2.6/utils/extdgts/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -11,7 +11,7 @@
 CC= @CC@
 DEFS= @DEFS@
 DEFINES= $(INCLUDES) $(DEFS) -D__USE_FIXED_PROTOTYPES_
-CFLAGS= -O $(DEFINES) 
+CFLAGS= @CFLAGS@ $(DEFINES) 
 PROGNAME=extdgts
 NAMES="../../bin/names.txt"
 
diff -Nur wwwcount2.6.orig/utils/mkstrip/Makefile.in wwwcount2.6/utils/mkstrip/Makefile.in
--- wwwcount2.6.orig/utils/mkstrip/Makefile.in	Sat Sep  4 05:16:40 1999
+++ wwwcount2.6/utils/mkstrip/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -11,7 +11,7 @@
 CC= @CC@
 DEFS= @DEFS@
 DEFINES= $(INCLUDES) $(DEFS) -D__USE_FIXED_PROTOTYPES_
-CFLAGS= -O $(DEFINES) 
+CFLAGS= @CFLAGS@ $(DEFINES) 
 PROGNAME=mkstrip
 NAMES="../../bin/names.txt"
 
diff -Nur wwwcount2.6.orig/utils/mwhich/Makefile.in wwwcount2.6/utils/mwhich/Makefile.in
--- wwwcount2.6.orig/utils/mwhich/Makefile.in	Thu Sep  2 22:07:54 1999
+++ wwwcount2.6/utils/mwhich/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -2,7 +2,7 @@
 CC= @CC@
 DEFS= @DEFS@
 DEFINES= $(INCLUDES) $(DEFS) -D__USE_FIXED_PROTOTYPES_
-CFLAGS= -O $(DEFINES) 
+CFLAGS= @CFLAGS@ $(DEFINES) 
 PROGNAME=mwhich
 
 LIBS=
diff -Nur wwwcount2.6.orig/utils/rgbtxt2db/Makefile.in wwwcount2.6/utils/rgbtxt2db/Makefile.in
--- wwwcount2.6.orig/utils/rgbtxt2db/Makefile.in	Sun Aug 22 04:51:24 1999
+++ wwwcount2.6/utils/rgbtxt2db/Makefile.in	Sun Jan  7 01:23:11 2001
@@ -19,7 +19,7 @@
 # replace -O with -g in order to debug
 
 DEFINES= $(INCLUDES) $(DEFS) -DSYS_UNIX=1 $(LDBMBACKEND)
-CFLAGS= -g $(DEFINES) $(LDBM_INCLUDES)
+CFLAGS= @CFLAGS@ $(DEFINES) $(LDBM_INCLUDES)
 
 SRCS = rgbtxt2db.c
 OBJS = rgbtxt2db.o
